results/intfloat__multilingual-e5-small/0a68dcd3dad5b4962a78daa930087728292b241d/SensitiveTopicsClassification.json

```diff
@@ -0,0 +1,73 @@
+{
+  "dataset_revision": "416b34a802308eac30e4192afc0ff99bb8dcc7f2",
+  "evaluation_time": 5.589444875717163,
+  "kg_co2_emissions": null,
+  "mteb_version": "1.11.6",
+  "scores": {
+    "test": [
+      {
+        "accuracy": 0.24384765625,
+        "f1": 0.20448955314534628,
```

Comment:
Scores on this task seem almost the same for the two models. Do you mind running the e5 base so see if we see a difference on this task?

Reply:
Yes, there it is:

| model                                                        |   SensitiveTopicsClassification |
|--------------------------------------------------------------|---------------------------------|
| untrained-multilingual                                       |                           17.79 |
| cointegrated__rubert-tiny2                                   |                           22.03 |
| sentence-transformers__paraphrase-multilingual-MiniLM-L12-v2 |                           24.84 |
| intfloat__multilingual-e5-small                              |                           24.38 |
| intfloat__multilingual-e5-base                               |                           24.98 |
| intfloat__multilingual-e5-large                              |                           26.07 |
| intfloat__multilingual-e5-large-instruct                     |                           29.48 |